app/controllers/admin/tasks_controller.rb

```diff
@@ -25,6 +26,7 @@ def create
   end
 
   def edit
+    @matching_evaluator = Evaluator.matching_task(@task)
```

Comment:
The name `Evaluator.matchin_task(@task)` is misleading. We can change it to `@task.compatible_evaluators`

```ruby
@evaluators = @task.compatible_evaluators
```

The change to `@evaluators` is because the view should render options to select - it does not care how the list was created; filtering is a controller concern.